### DISCUSSION TITLE
Can GIF auto-export after record stopped?

### DISCUSSION THREAD

**Discussion by MattL-V** [NickeManarin/ScreenToGif#1308] [Q&A]
Hi! I see that individual GIF frames are auto-saved as .PNG
After record finished, can they auto-save as an exported .GIF with default export setting and default name?

It would massively save time, rather than being forced to open the editor every time and manually exporting the GIF.

PS, It would also be great if the folder with the auto-exported GIF would instantly pop up after the record is over.

**Comment by jokerfyy** [comment]
收到啦，请放心，我马上就会看的
